msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -21,6 +21,54 @@ The Azure DevOps message extension uses link unfurling to look for URLs pasted i
 
 :::image type="content" source="~/assets/images/compose-extensions/messagingextensions_linkunfurling.png" alt-text="Example of link unfurling":::
 
+## App less adaptive card preview with schema.org templates
+
+Schema.org is a collaborative, community based activity with a mission to create, maintain, and promote schemas for the structured data on the internet, web pages, email and beyond.
+
+### How does it work
+
+When metadata of the webpage is tagged to a type of product it unfurls to an adaptive card. App type plays an crucial role, each type will have a designated template. The adaptive card is upgraded with lot more action buttons
+
+:::image type="content" source="../../assets/images/messaging-extension/meta.png" alt-text="Meta data"border="true"lightbox="../../assets/images/messaging-extension/meta-data.png":::
+
+### Adding meta data to the website
+
+Add structured schema.org JSON-LD metadata to the website. Few of the templates with @types:
```

Comment:
If JSON-LD is the only format that we support, then this is worth emphasizing. There are multiple ways to add markup to a site (see https://schema.org/docs/gs.html).

Reply:
update new steps.